src/assets/scripts/client/tutorial/TutorialView.js

```diff
@@ -283,9 +282,9 @@ export default class TutorialView {
         this.tutorial_step({
             title: 'Takeoff, part 1',
             text: ['When it appears at the start of runway ({RUNWAY}) (which may take a couple of seconds), click it (or press the up arrow once)',
-                   'and type in &lsquo;caf&rsquo; (for &lsquo;cleared as filed&rsquo;). This tells the aircraft it is cleared to follow its flightplan.',
+                   'and type in &lsquo;caf&rsquo; (for "cleared as filed"). This tells the aircraft it is cleared to follow its flightplan.',
```

Comment:
if we're escaping quotations and not encoding them, we should be consistent in how we do that.  these should be escaped double quotes.